python/lsst/ts/MTDomeTrajectory/dome_trajectory.py

```diff
@@ -338,17 +317,14 @@ async def move_dome_azimuth(self, desired_dome_azimuth):
 
             # Move the dome azimuth axis and wait for the target event
             # and the first motion event (so motion has started).
-            self.log.debug("Move dome azimuth.")
-            if not flushed_motion_event:
-                self.dome_remote.evt_azMotion.flush()
             self.dome_remote.evt_azTarget.flush()
+            self.log.debug("Start a dome azimuth move")
```

Comment:
Would you be willing to consider making the log message "Start dome azimuth motion" to be in line with line 310?